src/syscheckd/run_check.c

```diff
@@ -830,10 +848,6 @@ STATIC void fim_link_reload_broken_link(char *path, int index) {
 #endif
 #ifdef WIN_WHODATA
 void set_whodata_mode_changes() {
-    if (!syscheck.realtime) {
-        realtime_start();
```

Comment:
Are we certain we don't need to initialize realtime here? It looks to me that it could be possible to attempt to change whodata directories to realtime when no other directory is in this moda and could cause some trouble.